Wikipedia/Legacy Data Migration/WMFLegacyImageDataMigration.swift

```diff
@@ -23,20 +23,8 @@ public class WMFLegacyImageDataMigration : NSObject {
     /// Image controller where data will be migrated.
     let imageController: WMFImageController
 
-    /// List of saved pages which is saved when the tasks are finished processing.
```

Comment:
right, since it's now using the main saved page list, i have to access it on the main queue, meaning there's no need for the internal queue or `thenOn` etc.